tests/test_submit.py

```
@@ -0,0 +1,127 @@
+                }
+            ],
+        }
+        with open(self.json_file, "w") as f:
+            json.dump(submission_dict, f)
+
+    def tearDown(self) -> None:
+        shutil.rmtree(self.tmpdir)
+
+    def test_submit_help(self) -> None:
+        """Test dpdisp submit --help."""
+        sp.check_output(["dpdisp", "submit", "-h"])  # noqa: S603, S607
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (unused: `S603`)

Remove unused `noqa` directive

(RUF100)